src/app.module.ts

```diff
@@ -4,6 +4,8 @@ import { TypeOrmModule } from '@nestjs/typeorm';
 import { typeOrmConfig } from './config/database.config';
 import { TripModule } from './trip/trip.module';
 import { GoogleMapsModule } from './google-maps/google-maps.module';
+import { StatisticModule } from './statistic/statistic.module';
+import { StatisticController } from './statistic/statistic.controller';
```

Comment:
this should not be there

Reply:
I agree about "StatisticController" but "StatisticModule" must stay because it won't work